src/xcode/ENA/ENA/Resources/Localization/ro.lproj/Localizable.strings

```diff
@@ -655,30 +655,20 @@
 
 "Home_Risk_Inactive_Outdated_Results_Body" = "Înregistrarea în jurnal a expunerilor dvs. nu a putut fi actualizată timp de peste 24 de ore.";
 
-"Home_Risk_Inactive_Activate_Item_Title" = "Ultima înregistrare în jurnal a expunerilor: %@";
-
-"Home_Risk_Inactive_Activate_Item_Unknown_Title" = "Risc necunoscut";
-
-"Home_Risk_Inactive_Activate_Item_Low_Title" = "Risc redus";
-
-"Home_Risk_Inactive_Activate_Item_High_Title" = "Risc crescut";
-
 "Home_Risk_Inactive_NoCalculation_Possible_Button" = "Activați înregistrarea în jurnal a expunerilor";
 
 "Home_Risk_Inactive_Outdated_Results_Button" = "Activați înregistrarea în jurnal a expunerilor";
 
 /* Home Low card */
 "Home_Risk_Low_Title" = "Risc redus";
 
-"Home_Risk_Low_SaveDays_Item_Title" = "În ultimele %1$u zile, înregistrarea în jurnal a expunerilor a fost activă timp de %2$u zile.";
+"Home_Risk_Low_SaveDays_Item_Title" = "În ultimele 14 zile, înregistrarea în jurnal a expunerilor a fost activă timp de %1$u zile.";
```

Comment:
This as already corrected in 1.2.0 in stead of the 14 there should be the placeholder %2$u
See: https://github.com/corona-warn-app/cwa-app-ios/pull/1018/files